dotcom-rendering/src/components/Lightbox.stories.tsx

```diff
@@ -44,6 +45,7 @@ const Initialise = ({ children }: { children: React.ReactNode }) => {
 		const lightbox =
 			document.querySelector<HTMLDialogElement>('#gu-lightbox');
 		lightbox?.removeAttribute('hidden');
+		storage.local.clear();
```

Comment:
All stories can be affected by localstorage, so clearing localstorage here instead of only in the interaction story seems safer.